lib/sth.js

```diff
@@ -84,31 +90,34 @@ function exitGracefully(err, callback) {
  */
 function startup(callback) {
   if (isStarted) {
-    return process.nextTick(callback);
+    if (callback) {
+      return process.nextTick(callback);
+    }
+    return;
   }
 
   var version = sthHelper.getVersion();
   sthLogger.info(
-    'Starting up STH server version %s...',
-    version.version,
     {
-      operationType: sthConfig.OPERATION_TYPE.SERVER_START
-    }
+      op: sthConfig.OPERATION_TYPE.SERVER_START
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-sth-comet/commit/1f9ae996f0f354070de2ec9e8325ccb3831a1326